operational/flexera/custom_branding/flexera_custom_branding.pt

```diff
@@ -0,0 +1,359 @@
+  default "No Change"
+end
+
+parameter "param_marketing_footer_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Footer Hide"
+  description "A boolean to determine show/hide of the marketing footer on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_submit_ideas_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Submit Ideas Hide"
+  description "A boolean to determine show/hide of the Submit Ideas link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_open_new_case_link" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Open New Case Link"
+  description "A URL override pointing to your own Open New Case link on the landing page."
+  default "No Change"
+end
+
+parameter "param_marketing_support_help_link" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Support Help Link"
+  description "A URL override pointing to your own Support Help link on the landing page."
+  default "No Change"
+end
+
+parameter "param_marketing_support_plan_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Support Plan Hide"
+  description "A boolean to determine show/hide of the Support Plan Information link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_flexera_com_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Submit Ideas Hide"
+  description "A boolean to determine show/hide of the Flexera.com link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_login_logo_image" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Login Logo Image"
+  description "The web address of the image resource file location to use in place of the Flexera logo for unauthenticated pages like login, password reset, and so on."
+  default "No Change"
+end
+
+parameter "param_global_customer_support_link" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Global Customer Support Link"
+  description "A URL override pointing to your own Customer Support link on the login page and elsewhere throughout the app."
+  default "No Change"
+end
+
+parameter "param_marketing_login_image" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login Image"
+  description "The web address of the image resource file location to use in place of the Flexera marketing section on the login page."
+  default "No Change"
+end
+
+parameter "param_marketing_login_cta_link" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login CTA Link"
+  description "A URL override pointing to your own Download Now call to action link on the login page."

```

Comment:
Description should be a bit more clear. Always think in terms of a Finance person who had FinOps thrown at them but has little technical background when phrasing things in descriptions, parameters, etc.